mrmustard/physics/triples.py

```diff
@@ -252,19 +252,19 @@ def gket_state_Abc(symplectic: RealMatrix):
     return A[:m, :m], b, c
 
 
-def gdm_state_Abc(betas: Vector, symplectic: RealMatrix):
+def gdm_state_Abc(beta: float, symplectic: RealMatrix):
```

Comment:
I don't think it makes sense to restrict beta to be a float here. The generic gaussian density matrix can have different temperature in different modes. This is not a parallelizable state, it's usually entangled (unless the symplectic acts unitarily in each mode).